demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -341,7 +370,7 @@ def run_validations_on_pack(self, pack_path):
             else:
                 self.ignored_files.add(content_entity_path)
 
-        return all(pack_entities_validation_results)
+        return all(pack_entities_validation_results), FOUND_FILES_AND_ERRORS
```

Comment:
due to the change in return value (bool to tuple), please make sure there are no other calls to this function as they may break.

Reply:
Done